Examples/test-suite/arrays_dimensionless.i

```diff
@@ -9,10 +9,14 @@
 int globalints[] = {100, 200, 300};
 const int constglobalints[] = {400, 500, 600};
 
+class CC {};
+
 struct Bar {
     static int ints[];
+    static CC ccs[];
 };
 int Bar::ints[] = {700, 800, 900};
+CC Bar::ccs[] = {CC(), CC()};
```

Comment:
Just noting that this isn't specific to the new backend.  It looks OK to me, but highlighting for anyone else reviewing.

Reply:
This was added back in a2dc2756c (Several major fixes for: arrays, static members, member func.ptrs., exceptions, ... Lots of tests runs ok now., 2009-04-13) for completely unknown reasons. I didn't revert it because I didn't see any special reason to do it, but I don't have any special reason to keep it neither and could revert it if desired.